test/e2e/capi_test.go

```diff
@@ -58,11 +58,6 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 		Expect(e2eConfig.Variables).To(HaveKey(capi_e2e.KubernetesVersionUpgradeFrom))
 		v, err := semver.ParseTolerant(e2eConfig.GetVariable(capi_e2e.KubernetesVersionUpgradeFrom))
 		Expect(err).NotTo(HaveOccurred())
-		// Opt into Windows for versions greater than or equal to 1.22
-		if v.GTE(semver.MustParse("1.22.0")) {
-			Expect(os.Setenv("WINDOWS_WORKER_MACHINE_COUNT", "2")).To(Succeed())
```

Comment:
should we always set `WINDOWS_WORKER_MACHINE_COUNT` now?

Reply:
otherwise we won't be testing windows nodes because it defaults to 0 AFAIK

Reply:
Should that come in as a separate PR?

Reply:
oh i see, yea, we should probably just always set it.
Will update